### PR TITLE
controlplane queries

### PR DESCRIPTION
**Pull request by vineethvanga18**
Signed-off-by: vineethvanga18 <reddy.8@iitj.ac.in>

**Description**

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you, Vineeth.

**Comment by kumarabd** [comment]
@vineethvanga18 There is one that is already merged for this query. can you look that up once?

**Comment by vineethvanga18** [comment]
@kumarabd I dont think there is one for this(control plane). Although there are queries written for addons and operator status.

File: ui/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,90 @@
+enum AddonSelector {
```
Comment (dhruv0000) [inline ui/schema.graphql:1]: Why do we need one more schema file here @vineethvanga18 ? 
`    "relay": "relay-compiler --src ./components --schema ../internal/graphql/schema/schema.graphqls" ` uses the schema in the server itself.

Comment (vineethvanga18) [reply ui/schema.graphql:1]: It was already present. Though you are right.

Comment (leecalcote) [reply ui/schema.graphql:1]: So... what do we want to do here?

Comment (vineethvanga18) [reply ui/schema.graphql:1]: Just delete this file and use what is there at 'internal/graphql/schema' directory

Comment (leecalcote) [reply ui/schema.graphql:1]: Call for a re-review when this is done, please.

**Comment by dhruv0000** [comment]
image
@vineethvanga18 I think the resolves are in place, but the logic is still to be implemented. Maybe also use the http://localhost:9081/api/mesh/scan endpoint as an initial query for now?

**Comment by vineethvanga18** [comment]
@leecalcote we will use the old end point 'api/mesh/scan' for now until the resolvers are written for control plane. Although the query and subscription are done.

**Review by dhruv0000 — APPROVED** [review]

**Comment by leecalcote** [comment]
Ok, gents. Let's give this a go.

**Pull request merged**